contracts/seed/Seed.sol

```diff
@@ -316,32 +306,46 @@ contract Seed {
     /**
       * @dev                     Close distribution.
     */
-    function close() public onlyAdmin isActive {
+    function close() external onlyAdmin isActive {
+        // close seed token distribution
+        require(!minimumReached, "Seed: cannot close after minimum target is reached");
```

Comment:
Got questions with this line, can we discuss?  @codebuster22

Reply:
Yes, I'll ping you when I'm ready.

Reply:
Seems like the admin should be able to close even if paused.

Should allowed to close if and only if:

1) minimumReached and there are no seeds remaining to be claimed, or
2) !minimumReached and all funding tokens have been retrieved

Reply:
Yup, removed the require statement for `!paused`.

Reply:
Implemented
- Admin can close seed anytime.

Once Seed is closed, i.e. `closed == true`
- if `minimumReached == false`, admin can get back all the seeds that were transferred to `Seed` contract by invoking `retrieveSeedTokens`.
- else, admin can get back `seeds` that were not bought/distributed. 
  i.e. `seedToken.balanceOf( address(this) ) - totalSeedDistributed` 
  where `totalSeedDistributed = (seedAmountRequired + feeAmountRequired) - (seedRemainder + feeRemainder)`

Reply:
OK, so apparently we have changed the definition of "closed" to allow users to retrieve their seed and funding tokens at any time.  That means the admin can call `close` at any time.